MusiciansAPP/MusiciansAPP.API/Controllers/ArtistsController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using MusiciansAPP.API.Resources;
+using MusiciansAPP.API.Utils;
+using MusiciansAPP.BL.ArtistsService.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace MusiciansAPP.API.Controllers
+{
+    [Route("api/artists")]
+    [ApiController]
+    public class ArtistsController : ControllerBase
+    {
+        private readonly ILogger<ArtistsController> _logger;
+        private readonly IArtistsService _artistsService;
+        private readonly IMapper _mapper;
+
+        public ArtistsController(ILogger<ArtistsController> logger,
+            IArtistsService artistsService, IMapper mapper)
+        {
+            _logger = logger;
+            _artistsService = artistsService;
+            _mapper = mapper;
+        }
+
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<ArtistDto>>> GetTopArtists(
+            int pageSize, int page = 1)
```

Comment:
How does this parameter get here?
I don't see it in the route. And I didn't find it in the front-end service.